### PR TITLE
Closes #431: Fix back navigation in token management screen

### LINKED ISSUE #431: Going back arrow in token section doesn't work
**Describe the bug**
When being at token section and trying to go to the previous page, the going back button doesn't work.

**To Reproduce**
1. Go to gnosis safe
2. Click on add token at the main page.
3. Tap on the going back arrow in the upper left corner
4. Observe

**Expected behavior**
The page go back to the main menu 

**Actual behavior**
The arrow doesn't go back, it just refresh the token section.

**Smartphone (please complete the following information):**
 - Device: Motog6
 - Android Version: Android 8.0
 - App Version 1.6.1

### PR DESCRIPTION
**Pull request by elgatovital**
Closes #431.

@gnosis/mobile-devs

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: android
+dist: precise
```
Comment (tschubotz) [inline .travis.yml]: Is this change intended?

Comment (elgatovital) [reply .travis.yml]: yes, travis stoped working because default image lacks android tools
